roaring/roaring.go

```diff
@@ -127,20 +167,13 @@ func (b *Bitmap) Add(a ...uint64) (changed bool, err error) {
 
 func (b *Bitmap) add(v uint64) bool {
 	hb := highbits(v)
-	i := search64(b.keys, hb)
-
-	// If index is negative then there's not an exact match
-	// and a container needs to be added.
-	if i < 0 {
-		b.insertAt(hb, newContainer(), -i-1)
-		i = -i - 1
-	}
-	return b.containers[i].add(lowbits(v))
+	cont := b.conts.GetOrCreate(hb)
```

Comment:
for consistency with the next line, this one could be `cont := b.conts.GetOrCreate(highbits(v))`

Reply:
why recalculate again if the value is already saved from above?